src/modules/vendor/components/courses/CourseCard.vue

```diff
@@ -75,13 +89,22 @@ export default {
     min-height: auto;
     margin-right: 10px;
   .infos-course-container
-    display: flex;
+    align-items: center;
     font-size: 14px;
     margin-top: 10px;
-  .infos-course-container > .q-item__section
-    display: flex;
-    justify-content: flex-start;
-    flex-direction: row;
-  .infos-course-container > .q-item__section > .q-icon
-    margin-right: 10px;
+    & > .q-item__section
+      display: flex;
+      justify-content: flex-start;
+      flex-direction: row;
+      color: $black;
+      &.q-item__section--side
+        margin-right: 10px
+      & > .q-icon
+        margin-right: 10px;
+  .slots
+    height: 10px;
+    flex: 0 1 calc(25% - 4px);
```

Comment:
je ne m'y connais pas trop en ecriture flex comme ca : tu peux m'expliquer chaque element stp ?

Reply:
en gros, la propriété flex s'utilise sur les éléments d'un container flexbox et chaque valeur correspond à (dans l'ordre): 
- flex-grow -> par défaut, elle est à zéro et correspond à la place que peut prendre un élément ds le container en fonction de l'espace qu'il reste.
- flex-shrink -> par défaut, elle est à 'auto' et c'est en fait l'inverse de flex-grow car elle agit sur le rétrécissement de l'élément.
- flex-basis -> par défaut, elle est à 'auto' et elle permet de définir une longueur pour chaque élément ds la flexbox. j'ai ici utilisé la fonction css `calc` (merci stackoverflow :) ) pour que la longueur prenne en compte l'espace du gutter horizontal soit 4px.

flexbox cheatsheet ^^: https://css-tricks.com/snippets/css/a-guide-to-flexbox/